README.md

```diff
@@ -19,1028 +17,1075 @@ lacking thorough examples, but some things in the RSpec world change week by
 week or month by month. With that said, as the standard changes this guide is
 meant to be able to change with it.
 
-## Style Guide Rules
+## Table of Contents
 
+* [Layout](#layout)
+* [Example Structure](#example-structure)
+* [Naming](#naming)
+* [Matchers](#matchers)
+* [Views](#views)
+* [Controllers](#controllers)
+* [Models](#models)
+* [Mailers](#mailers)
 
-### Line Returns after `feature`, `context`, or `describe`
+## Layout
 
-Do not leave line returns after `feature`, `context` or `describe`
-descriptions. It does makes the code more difficult to read and lowers the
-value of logical chunks.
+* <a name="empty-lines-after-describe"></a>
+Empty lines after `feature`, `context`, or `describe`
+<sup>[[link](#empty-lines-after-describe)]</sup>
 
-#### Bad Example
+  Do not leave empty lines after `feature`, `context` or `describe`
+  descriptions. It does makes the code more difficult to read and lowers the
+  value of logical chunks.
 
-```ruby
-describe Article do
+  ```ruby
+  # bad
+  describe Article do
 
-  describe '#summary' do
+    describe '#summary' do
 
-    context 'when there is a summary' do
+      context 'when there is a summary' do
 
-      it 'returns the summary' do
-        # ...
+        it 'returns the summary' do
+          # ...
+        end
       end
     end
   end
-end
-```
 
-#### Good Example
-
-```ruby
-describe Article do
-  describe '#summary' do
-    context 'when there is a summary' do
-      it 'returns the summary' do
-        # ...
+  # good
+  describe Article do
+    describe '#summary' do
+      context 'when there is a summary' do
+        it 'returns the summary' do
+          # ...
+        end
       end
     end
   end
-end
-```
+  ```
+
+* <a name="empty-lines-after-let"></a>
+Empty lines after `let`, `subject`, and `before`/`after`
```

Comment:
In the other guides normally the entire rule is the anchor (or at least a complete sentence as a summary), but I call live with those "meta headings".